Improve compatibility between pipeline & cluster (#21)
VIB-PSB/ksrates: doc/source/usage.rst

The *ksrates* pipeline can be automatically run through Nextflow with a few preparation steps.

1.  Access in a terminal the directory that will host the rate-adjustment results (assumed here to be ``example``) and unzip the sequence data files in there::

        cd ksrates/example
        gunzip elaeis.fasta.gz oryza.fasta.gz asparagus.fasta.gz elaeis.gff3.gz

2.  Prepare the configuration files.

>     The directory already contains a pre-filled *ksrates configuration file* (``config_elaeis.txt``) and a *Nextflow configuration file* template (``custom_nextflow.config``) to be filled in as described in the :ref:`nextflow_config_section` section.

Reviewer: Would it make sense to rename the custom_nextflow.config to simply the default nextflow.config? Otherwise we should add the "-c custom_nextflow.config" to all nextflow command lines everywhere in the documentation (e.g. also in Installation -> Testing your installation).

Author: Now that it is a "mandatory" file it makes much sense to do that. By renaming it just nextflow.config it will be automatically detected and used, and we can drop the "-c" flag in the launching command.
Shall I delete it from everywhere then? I'll just note down in the Usage that you can explicitly give the path with "-c" if you want to call your file another way or if you want to give a specific path.

Reviewer: Sounds good to me, just make sure that it's still described that it is needed and should be called nextflow.config.

3.  Launch *ksrates* through the following command line::

        nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt

    The *ksrates configuration file* is provided through the ``--config`` option. The *Nextflow configuration file* is automatically recognized when it's named with the Nextflow-reserved ``nextflow.config`` filename and it's located in the launching directory; alternatively, the user can provide a costum filename or path through the ``-c`` option.
    

>     The first time the command is launched it downloads the *ksrates* Nextflow pipeline from the ``VIB-PSB/ksrates`` GitHub repository; from then on it uses the local copy stored in the ``.nextflow`` directory. If running a container, the image is pulled from Docker Hub and stored locally for successive usage.

Author: Yes, it's in home, just checked. I'll add it!

        nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt -c ./custom_nextflow.config

>     The ``--config`` option takes the *ksrates configuration file*, while ``-c`` takes the *Nextflow configuration file*. If the Nextflow-reserved ``nextflow.config`` name is used, this latter file is automatically recognized without explicitly calling it in the command line.

Reviewer: ```suggestion
    The ``--config`` option takes the *ksrates configuration file*, while ``-c`` takes the *Nextflow configuration file*. If the Nextflow-reserved ``nextflow.config``file name is used for the latter, the file is automatically recognized without having explicitly configure it with the ``-c`` option.
```
